package.json

```diff
@@ -94,15 +94,18 @@
   },
   "homepage": "https://threejs.org/",
   "devDependencies": {
+    "@dimforge/rapier3d-compat": "^0.17.3",
     "@rollup/plugin-node-resolve": "^16.0.0",
     "@rollup/plugin-terser": "^0.4.0",
+    "ammojs3": "^0.0.11",
     "concurrently": "^9.0.0",
     "eslint": "^8.37.0",
     "eslint-config-mdcs": "^5.0.0",
     "eslint-plugin-compat": "^6.0.0",
     "eslint-plugin-html": "^8.0.0",
     "eslint-plugin-import": "^2.27.5",
     "jimp": "^1.6.0",
+    "jolt-physics": "^0.23.0",
```

Comment:
For security reasons, we are trying to keep the dev dependencies to a minimum. Just adding the dependencies to get type information is no sufficient reasons for enhancing `package.json`.

Reply:
I'm afraid I can reopen the PR, see :

<img width="402" height="90" alt="image" src="https://github.com/user-attachments/assets/c03f04a4-cda1-4603-b319-99d836b52936" />